web-app/src/components/Legend/container.js

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import { PropTypes as PT } from 'prop-types';
-import holidayStatus from '../../utilities/holidayStatus';
+import _ from 'lodash';
```

Comment:
@paul-matchett You're importing the entirety of lodash here. Import only what you need. This may be because of the chain method you're using so give this a read:
https://medium.com/making-internets/why-using-chain-is-a-mistake-9bc1f80d51ba